website/docs/sealed.md

```diff
@@ -100,6 +100,16 @@ else T.absurd(x)                     # error: Missing case for `Failed`
 end
 ```
 
+As an extra convenience, a sealed class or module will also have a method called
+`sealed_subclasses` that returns the complete set of sealed subclasses. This can
+be used to dynamically iterate over all the subclasses:
+
+```ruby
+ChargeAttemptResult.sealed_subclasses.each do |klass|
+  puts "Charge attempt can be #{klass.name}"
+end
+```
+
```

Comment:
Would you mind writing this in a separate section? I'm sure I'll eventually want to anchor link to it.

Reply:
Also since it's a set, can we not encourage people to iterate—the canonical operation on a set is `include?`, and we probably don't want to encourage people to iterate because the iteration order isn't guaranteed.